src/components/graph-view.js

```diff
@@ -260,8 +261,7 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
       nextProps.edges !== this.props.edges ||
       !nextState.componentUpToDate ||
       nextProps.selected !== this.props.selected ||
-      nextProps.readOnly !== this.props.readOnly ||
-      nextProps.layoutEngine !== this.props.layoutEngine
```

Comment:
why did you remove this layoutEngine check here?

Reply:
Complete accident, I think when I was doing clean up I accidentally deleted too much